ports/esp32s2/README.md

```diff
@@ -48,6 +48,9 @@ Building boards such as the Saola is typically done through `make flash`. The de
 make BOARD=espressif_saola_1_wrover flash PORT=/dev/tty.usbserial-1421120
 ```
 
+More information about the `esp-idf` can be found [here](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/).
```

Comment:
Small suggestion, I think a link to `esp32s2` page in `esp-idf` would be more relevant. 

```suggestion
More information about the `esp-idf` can be found [here](https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/).
```